manifests/10_operator.yaml

```diff
@@ -47,3 +47,6 @@ spec:
                   fieldPath: metadata.namespace
             - name: OPERATOR_NAME
               value: "marketplace-operator"
+            # The string "0.0.1-snapshot" is substituted by the CVO with the version of the payload
```

Comment:
Please move the commit to the correct spot

Reply:
Will do